etc/proto/protoc-gen-zap/gogoproto/gogo.pb.go

```diff
@@ -0,0 +1,1298 @@
+// Protocol Buffers for Go with Gadgets
```

Comment:
Does this grow our dependency on gogo?  I thought we were trying to reduce that.  Otherwise it looks like the new stuff depends on the official go protobuf packages, which is good.

Reply:
Yeah, this is the gogo proto generated with the new protobuf compiler/library.  Necessary to read the annotations from the new protobuf api (they get registered in a global data structure at import time and standard/gogo use different places :/).  When we stop using gogo, we just delete this file, remove the references in main.go to gogoproto.E_Customname and friends, and it's gone.